docs/Acceptable-Formulae.md

```diff
@@ -73,6 +73,12 @@ Clang is the default C/C++ compiler on macOS (and has been for a long time). Sof
 
 We're a package manager so we want to do things like resolve dependencies and set up applications for our users. If things require too much manual intervention then they aren't useful in a package manager.
 
+### Static libraries
+
+In general, packages should not ship static libraries since these cannot be updated without a rebuild of the dependant software.
+If a package gets a lot of requests to provide static libraries, they may be provided by the package.
```

Comment:
```suggestion
In general, formulae should not ship static libraries since these cannot be updated without a rebuild of the dependant software.
If a formula gets a lot of requests to install static libraries, they may be installed by the formula.
```